### PR TITLE
#2243 - Fix markdown <code> element issue

### PR DESCRIPTION
**Pull request by SebinSong**
#2243 

[ Fix screenshot ]

### CODE REVIEW

**Comment by taoeffect** [comment]
Question, the issue solution section says:

> When text goes beyond that width, a horizontal scrollbar should appear at the bottom of the codeblock (don't wrap the text, in other words)

But the screenshot shows the text being wrapped. The reason why I wrote add a horizontal scrollbar is because wrapping text sometimes makes it hard to read source code on a small screen and be able to tell what line is what. Do you think you could update this to use the horizontal scroll bar?

**Comment by SebinSong** [comment]
@taoeffect Updated again with the fix

File: frontend/views/containers/chatroom/MessageReactions.vue [hunk 1]
```diff
@@ -82,6 +82,7 @@ export default ({
 .c-emoticons-list {
   padding-left: 3rem;
   display: flex;
+  flex-wrap: wrap;
```
Comment (SebinSong) [inline frontend/views/containers/chatroom/MessageReactions.vue:85]: This is a fix for the overflowing emoji issues @Silver-IT  found.

**Review by taoeffect — APPROVED** [review]
Looking really solid - great work @SebinSong !

**Pull request merged**